alerting/src/main/kotlin/org/opensearch/alerting/DocumentLevelMonitorRunner.kt

```diff
@@ -264,7 +264,14 @@ object DocumentLevelMonitorRunner : MonitorRunner() {
             monitorOrTriggerError = actionCtx.error
         )
 
-        for (action in trigger.actions) {
+        val actionsToBeProcessed = actionsToProcessInTrigger(trigger, MonitorRunnerService.monitorCtx.maxActionsPerTrigger)
+        if (trigger.actions.size > actionsToBeProcessed.size)
+            logger.warn(
+                "Some actions from trigger ${trigger.name} will not be processed as they would exceed the maximum" +
+                    " amount of allowed actions per trigger."
+            )
```

Comment:
This logic should be in `actionsToProcessInTrigger` since its shared amongst all the monitor runners